go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/containers/common
 
-go 1.20
+go 1.21
+
+toolchain go1.22.1
```

Comment:
This should not have been merged, this sets the minimum version used to compile.
Also podman still uses go1.20 on fedora 38

I will revert this as it is blocking the vendor proccess